<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>1366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1221</width>
      <height>1271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_ajouter">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1221</width>
        <height>911</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout des Employee</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_matricule">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_Nom">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_prenom">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_GSM">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>GSM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Role">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Role</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Matricule">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Prenom">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Role">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>230</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_Salaire">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Salaire</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Nom">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mail">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Salaire">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>270</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_mail">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Mail</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_GSM">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>190</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>220</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>60</y>
         <width>561</width>
         <height>271</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>559</width>
          <height>269</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="options">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>561</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>options</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Afficher">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricule</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_employee">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1051</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Matricule_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>340</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricule</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Nom_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>370</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_prenom_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>400</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_mail_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>430</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_GSM_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>460</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>GSM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Role_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>490</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Role</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Salaire_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>520</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Salaire</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_matricule_update">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>340</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nom_update">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>370</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_salaire_update">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>520</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_role_update">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>490</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_gsm_update">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>460</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mail_update">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>430</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom_update">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>400</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_update">
      <property name="geometry">
       <rect>
        <x>372</x>
        <y>300</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>350</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>chercher les employes par taper leur coordonneés</string>
      </property>
     </widget>
     <widget class="QLabel" name="qr_code">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>580</y>
        <width>661</width>
        <height>471</height>
       </rect>
      </property>
      <property name="text">
       <string>qrCode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qrCode">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>830</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>QR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoadButton_Historique">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>310</y>
        <width>191</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="radioButton_tri_personnel_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string> Trier par nom</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_tri_personnel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string> Trier par gsm</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_tri_personnel_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string> Trier par Matricule</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="recherche_par_reference_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>400</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>historique.</string>
     </attribute>
     <widget class="QTableView" name="tableView_historique">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>921</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="homeButton_recherche">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>200</y>
        <width>171</width>
        <height>321</height>
       </rect>
      </property>
      <property name="text">
       <string>actualiser</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1313</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
